Survey/SurveyUITests/SurveyUITestsLaunchTests.swift

```diff
@@ -21,9 +14,6 @@ class SurveyUITestsLaunchTests: XCTestCase {
         let app = XCUIApplication()
         app.launch()
 
-        // Insert steps here to perform after app launch but before taking a screenshot,
```

Comment:
why remove the comments?

Reply:
Codacy.. I have removed all comments, otherwise I would have to count number of words in each component..